epxresults/job.py

```diff
@@ -110,6 +119,67 @@ def __init__(self, **kwargs) -> None:
         self.job_id = self._return_job_id(**kwargs)
         self.conditions, self.global_variables = self._parse_vars()
         self._snapshot_map = self._set_snapshot_map()
+        self._results = self._fred_results(**kwargs)
+
+    def _fred_results(self, **kwargs) -> Path:
+        """
+        The FRED results directory associated with the job
+
+        Notes
+        -----
+        If no results directory is associated with the job, `None` will be
+        returned.
+        """
+
+        if 'PATH_TO_JOB' in kwargs.keys():
+            root = Path(self.path_to_job).parent
+            while root != Path('/'):
+                if _is_valid_results_directory(root):
+                    return root
+                else:
+                    root = root.parent
+            return None
+        else:
+            return Path(_path_to_results(**kwargs))
+
+    def delete(self, force=False, verbose=False):
+        """
+        Delete the FRED job
+        """
+
+        if force is False:
+            permission_to_delete = None
+            q = (f"Are you sure you want to delete job {self.job_key}?")
+            while permission_to_delete is None:
+                reply = str(input(q + ' (y/n): ')).lower().strip()
+                if reply[0] == 'y':
+                    permission_to_delete = True
+                if reply[0] == 'n':
+                    permission_to_delete = False
```

Comment:
Is `FREDJob` intended to be used as a command-line interface rather than a library? My concern is that if someone tries to use `FREDJob.delete` in a script (e.g. `fred-cloud-execute`) but leaves `force=False` (the default), this will hang indefinitely waiting for interactive input that will never come.

Reply:
I use it in interactive python sessions, but primarily I intend it to be used in scripts.  I wanted to try and make the delete method a little safer to make sure users knew what they were doing.  But I agree.  **I** intend to use it with the `force` option set to `True` always.  Should we just remove this check and assume users know what they are doing?

Reply:
Yea I think it's clear what `FREDJob.delete` is going to do. If you're worried about accidentally running it in interactive sessions, you could add 

```python
class FREDJob
...
    def safety_on(self) -> None:
        self.permission_to_delete = False

   def safety_off(self) -> None:
        self.permission_to_delete = True
```
methods. Then after calling
```python
my_fred_job.safety_on()
```
You'd know you couldn't accidentally delete the job data. Seems pretty edge-case to me though.

Reply:
okay.  I will remove the overly cautious approach and let users delete jobs if they want.  We can add a safety in the future if we need it?